core/src/test/java/io/lionweb/lioncore/java/serialization/MyNodeWithProperties.java

```diff
@@ -33,8 +33,13 @@ public MyNodeWithProperties(String id) {
     super(id, CONCEPT);
   }
 
-  public boolean getP1() {
-    return (boolean) this.getPropertyValueByName("p1");
+  public Boolean getP1() {
+    Object value = this.getPropertyValueByName("p1");
+    if (value == null) {
```

Comment:
In test code, so not so important, but I find this logic better readable when using a ternary operator.

Reply:
Fixed